site/en/guide/tpu.ipynb

```diff
@@ -343,7 +348,7 @@
         "id": "8hSGBIYtUugJ"
       },
       "source": [
-        "To reduce python overhead, and maximize the performance of your TPU, try out the **experimental** `experimental_steps_per_execution` argument to `Model.compile`. Here it increases throughput by about 50%:"
+        "To reduce Python overhead and maximize the performance of your TPU, you can pass in the **experimental** argument—`experimental_steps_per_execution`—to `Model.compile`. In this example, it increases throughput by about 50%:"
```

Comment:
```suggestion
        "To reduce Python overhead and maximize the performance of your TPU, pass in the **experimental** argument—`experimental_steps_per_execution`—to `Model.compile`. In this example, it increases throughput by about 50%:"
```

While "you" is allowed, generally prefer to write so it's not needed. /Implicit you/, I suppose. Repeated use is a bit too friendly, imho.